src/card-composable/__tests__/__stories/card-composable.stories.tsx

```diff
@@ -151,6 +175,231 @@ const href = 'javascript:void(0);';
 const areasGap = 'space050';
 const contentGap = 'space040';
 
+// The Sun --- starts here
+const IconFilledVideoCam = toNewsKitIcon(FilledVideocam);
+const IconFilledShare = toNewsKitIcon(FilledShare);
+
+export interface MediaContextActionProps {
+  size?: 'small' | 'medium' | 'large';
+  area?: string;
+}
+
+const map = {
+  small: '375px',
+  medium: '640px',
+  large: '920px',
+};
+// The Sun --- ends here
+
+const typographyPresetMap = {
+  small: 'editorialDisplay003',
+  medium: 'editorialDisplay004',
+  large: 'editorialDisplay005',
+};
+
+const CardVerticalTheSun: React.FC<MediaContextActionProps> = ({
+  size = 'medium',
+}) => (
+  <CardComposable
+    overrides={{width: map[size]}}
+    style={{
+      overflow: 'hidden',
+      borderBottom: '1px solid #eaeeef',
+      borderLeft: '1px solid #eaeeef',
+      borderRight: '1px solid #eaeeef',
+    }}
+  >
+    <CardMedia columns="1fr" rows="1fr">
+      <GridLayoutItem column="1 / 2" row="1 / 2">
+        <Image
+          src={
+            size === 'small'
+              ? 'https://storybook.newskit.co.uk/placeholder-1x1.png'
+              : 'https://storybook.newskit.co.uk/placeholder-3x2.png'
+          }
+          alt="card-media-image"
+          loadingAspectRatio={size === 'small' ? '1:1' : '3:2'}
+        />
+      </GridLayoutItem>
+      <GridLayoutItem
+        column="1 / 2"
+        row="1 / 2"
+        alignSelf="end"
+        justifySelf="start"
+        style={{zIndex: 2}}
+        marginInlineStart="space040"
+      >
+        <Flag overrides={{typographyPreset: 'editorialLabel020'}}>
+          EXCLUSIVE
+        </Flag>
+      </GridLayoutItem>
+    </CardMedia>
```

Comment:
Sorry for the double comment but there is an even better way of that, in that way you don't change anything in the CardMedia, but use extra GridLayoutItem which uses same area name.
```suggestion
     <CardMedia
      media={{
        src:
          size === 'small'
            ? 'https://storybook.newskit.co.uk/placeholder-1x1.png'
            : 'https://storybook.newskit.co.uk/placeholder-3x2.png',

        alt: 'card-media-image',
        loadingAspectRatio: size === 'small' ? '1:1' : '3:2',
      }}
    />
    <GridLayoutItem
      area="media"
      alignSelf="end"
      justifySelf="start"
      style={{zIndex: 2}}
      marginInlineStart="space040"
    >
      <Flag overrides={{typographyPreset: 'editorialLabel020'}}>EXCLUSIVE</Flag>
    </GridLayoutItem>
```

Reply:
This is cool